tests/pass/tree-borrows/cell/cell-alternate-writes.rs

```diff
@@ -0,0 +1,27 @@
+//@compile-flags: -Zmiri-tree-borrows -Zmiri-tag-gc=0
+#[path = "../../../utils/mod.rs"]
+mod utils;
+use utils::macros::*;
```

Comment:
could also add a crate and make it a dependency of https://github.com/rust-lang/miri/blob/master/test_dependencies 

that way we can reuse it in all tests easily without having to fiddle with path imports